octobot/src/server/github_handler.rs

```diff
@@ -822,13 +838,7 @@ impl GithubEventHandler {
             return;
         }
 
-        if comment.user().login() == self.github_session.bot_name()
-            || self
-                .config
-                .slack
-                .ignored_users
-                .contains(&comment.user().login().to_string())
-        {
+        if comment.user().login() == self.github_session.bot_name() {
```

Comment:
kept the octobot silence just for comments since that's basically all it ever does. 🤷 

(i guess it does reapprove PRs sometimes, but that's pretty rare, and maybe worth the DM still)